helpers/npm/lib/updater.js

```diff
@@ -68,20 +68,17 @@ async function updateDependencyFiles(
 }
 
 function install_args(depName, desiredVersion, requirements, oldLockfile) {
-  const source = (
-    requirements.find(req => {
-      return req.source;
-    }) || {}
-  ).source;
+  const source = (requirements.find(req => req.source) || {}).source;
 
   if (source && source.type === "git") {
-    var originalVersion = ((oldLockfile["dependencies"] || {})[depName] || {})[
+    let originalVersion = ((oldLockfile["dependencies"] || {})[depName] || {})[
       "version"
     ];
-    originalVersion = originalVersion || `${source.url}#ref`;
+
     if (!originalVersion.includes("#")) {
```

Comment:
@ygnoh looks like `originalVersion` could be undefined here so `.includes()` will throw, could you guard against this here? `if (!originalVersion || !originalVersion.includes("#")) {...`

Reply:
Thank you! It would be `undefined`. I think `oldLockFile` always has some value, not `undefined`. I will change it!